enterprise/models.py

```diff
@@ -4941,3 +4941,48 @@ class OnboardingFlow(TimeStampedModel):
             "Specifies whether this flow is active and shown to admin users in the onboarding tour."
         )
     )
+
+
+class EnterpriseCustomerAdmin(TimeStampedModel):
+    """
+    Model for tracking enterprise customer admin users and their onboarding status.
+
+    .. no_pii:
+    """
+
+    uuid = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    enterprise_customer_user = models.ForeignKey(
+        EnterpriseCustomerUser,
+        blank=False,
+        null=False,
+        related_name='admin_records',
+        on_delete=models.deletion.CASCADE,
+        help_text=_("The enterprise customer user who is an admin.")
+    )
+    last_login = models.DateTimeField(
+        null=True,
+        blank=True,
+        help_text=_("The last time the admin logged into the admin portal.")
+    )
+    completed_tour_flows = JSONField(
```

Comment:
We might want to change this to a ManyToMany field so we can directly connect it to the OnboardingFlow model, like `completed_tour_flows = models.ManyToManyField(OnboardingFlow, null=True, blank=True)`